github_copilot_seats/seats.py

```diff
@@ -0,0 +1,113 @@
+import json
+import os
+from datetime import datetime, timezone, timedelta
+from typing import Any
+import autokitteh
+from autokitteh.github import github_client
+from autokitteh.slack import slack_client
+from helpers import github_username_to_slack_user_id
+
+GITHUB_ORG = os.getenv("GITHUB_ORG")
```

Comment:
If I'm not mistaken, all references to a GitHub org are now obsolete. We needed them only in the Starlark wrapper for GitHub, to distinguish authenticated API calls from anonymous ones.

Reply:
you're saying I can get the org from the python client? I need the org name to get the copilot object

Reply:
Oops, I missed `github.get_organization`. Anyway, yes, I think so. But feel free to check this out at a later time.